streaming/WORKSPACE

```diff
@@ -2,12 +2,12 @@ workspace(name = "com_github_ray_streaming")
 
 # LOAD RAY WORKSPACE
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
-ray_version = "2cee219250ccd8900ec8a7c1858310a42a7ca3ed"
+ray_version = "remove-exporting-symbols"
```

Comment:
We need change this version number.

Reply:
> We need change this version number.

I'll change this version to master in commid id.